ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/multiplechoice/MultipleChoiceTaskViewModel.kt

```diff
@@ -92,6 +92,12 @@ class MultipleChoiceTaskViewModel @Inject constructor() : AbstractTaskViewModel(
   }
 
   fun updateResponse() {
+    // Check if "other" text is missing or not.
+    if (task.isRequired && selectedIds.contains(OTHER_ID) && otherText.trim().isEmpty()) {
```

Comment:
Please extra into helper function. You can also use `isBlank()` here on `otherText`